pkg/runner/local_runner.go

```diff
@@ -136,6 +157,44 @@ func (r *localRunner) Run(ctx context.Context, request *runner.RunRequest) (*run
 	}
 	cmd.Stdout = stdout
 
+	// Mount special filesystems.
+	requiresSpecialFilesystems := cmd.SysProcAttr != nil && cmd.SysProcAttr.Chroot != ""
+	var MountRootDirectory filesystem.Directory
+
+	if requiresSpecialFilesystems {
+		var dir filesystem.Directory
+		dir = r.buildDirectory
+		// TODO(nils): How to best open the input root `Directory`
```

Comment:
This seems to be a misunderstanding on my part.
I want (to hold on to) a `Directory` object for the input root.
It can be reopened, but do not want to call this opening code twice, I don't think.

Reply:
Yeah, you likely also don't want it, because the directory can be moved around to other places, right?